client/src/__mocks__/gatsby.ts

```diff
@@ -1,9 +1,10 @@
-/* eslint-disable no-unused-vars */
-const React = require('react');
 
+/* eslint-disable no-unused-vars */
+import React from 'react'
 const gatsby = jest.requireActual('gatsby');
 
-const envData = require('../../../config/env.json');
+// @ts-ignore
```

Comment:
```suggestion
```
If you run `npm run create:config` locally, the `env.json` file will get created.  Since this happens before we typecheck the code, it's not necessary to ts-ignore it.

Reply:
okay, will update